### ISSUE TITLE
ICICI bank home page !the radio button under the tab ‘your net annual income ‘is not announce the value properly

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#3]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2..pressed H for heading for four time to reach the heading ‘104M+ Downloads’ 
.3 after tab two time select the next tab ‘find the right product for you’
4. after tab three time reached to the focus radio button. 
Expected Result under the tab of net annual income -the radio button option should be announced exact selected value. 

Actual Result .under the tab of ‘net annual income’ screen reader announce ‘ radio button checked one of four
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value
